components/user-details.vue

```diff
@@ -209,6 +223,22 @@
         return (this.user.affiliation === 'BUSINESS' && this.user.companyName != null)
       },
 
+      isRepeatCustomer () {
+        let count = 0
+        this.orders.forEach(element => {
+          if (this.orderId) {
+            if (element.status === 'shipped_complete' && this.orderId !== element.id) count++
+          } else {
+            if (element.status === 'shipped_complete') count++
+          }
+        })
+        if (count > 0) {
+          return true
+        } else {
+          return false
+        }
+      },
```

Comment:
We can still simplify this process a lot. We shouldn't take into consideration the current order because it makes this dependent on the ticket the agent is viewing. Take this example:

If a customer has an order that is fully shipped,
1) An agent views the order ticket and sees they _are not_ a repeat customer
2) The customer opens a sales ticket, an agent would see they _are_ a repeat customer


```suggestion
      isRepeatCustomer () {
        return this.orders.some(o => (o.status === 'shipped_complete'))
      },
```

Reply:
Following up for those reading along who aren't in Slack. We talked to Sam and concluded that the context of the current ticket is critical in determining whether or not someone is a return customer:

Customer A has a previous order with us.
Customer A places a new order.
Agent **should** see Customer A is a repeat customer because this new ticket (order verification) satisfies the "repeat" order checkbox.

Customer B has a previous order with us.
Customer B opens a sale question.
Agent **should not** the customer labeled as repeat, the customer only has 1 order and the current ticket does not pertain to an in-progress order.

Customer C has 2 previous orders with us.
Customer C opens a sale question.
Agent **should** see Customer C is a repeat customer because they have more than 2 completed orders.